package.json

```diff
@@ -27,6 +27,8 @@
     "mongoose": "^4.8.1",
```

Comment:
express-session is not listed under the dependencies in package.json. Remember that you have to install it using
`npm install --save express-session`